src/main/resources/assets/mythicmetals/lang/ko_kr.json

```diff
@@ -0,0 +1,819 @@
+  "text.config.mythicmetals.option.unobtainium.trapezoid": "언옵테이늄 사다리꼴 범위를 사용할까요?",
+  "text.config.mythicmetals.option.unobtainium.veinSize": "언옵테이늄 광맥 최대 크기",
+
+  "text.config.mythicmetals.title": "Mythic Metals 환경설정",
+
+  "subtitles.mythicmetals.equip_runite": "익숙한 갑옷이 철커덕 소리를 냄",
+
+  "itemGroup.mythicmetals.main": "Mythic Metals",
+  "itemGroup.mythicmetals.main.button.github": "Mythic Metals 이슈 트래커",
+  "itemGroup.mythicmetals.main.button.curseforge": "Mythic Metals Curseforge 페이지",
+  "itemGroup.mythicmetals.main.button.modrinth": "Mythic Metals Modrinth 페이지",
+  "itemGroup.mythicmetals.main.button.discord": "도움이 필요하신가요? Wraith Coding Sesh 디스코드 서버에 가입하여 도움을 받으세요.",
+  "itemGroup.mythicmetals.main.tab.items": "아이템",
+  "itemGroup.mythicmetals.main.tab.tools": "도구",
+  "itemGroup.mythicmetals.main.tab.blocks": "블록",
+  "itemGroup.mythicmetals.main.tab.armor": "방어구",
+
+  "advancement.mythicmetals.root.title": "Mythic Metals",
+  "advancement.mythicmetals.root.description": "깊은 지하에서 모든 광석을 발견하세요.",
+  "advancement.mythicmetals.adamantite.description": "심층암 동굴의 가장 깊은 곳에서 아다만타이트를 찾으세요.",
+  "advancement.mythicmetals.adamantite.title": "아이테르의 단조",
+  "advancement.mythicmetals.aquarium.description": "바다 밑에서 아쿠아리움을 찾으세요.",
+  "advancement.mythicmetals.aquarium.title": "수중 탐험",
+  "advancement.mythicmetals.aquarium_pearl.description": "삼지창을 만드는 데 사용할 수 있는 아쿠아리움 진주를 획득하세요.",
+  "advancement.mythicmetals.aquarium_pearl.title": "스페이스 드래곤처럼 반짝반짝 빛나는",
+  "advancement.mythicmetals.banglum.description": "다른 금속을 향한 첫 단계인 뱅그룸을 찾으세요.",
+  "advancement.mythicmetals.banglum.title": "전설의 뱅",
+  "advancement.mythicmetals.banglum_chunk.description": "뱅그룸 광석에서 거의 나오지 않는 뱅그룸 덩어리를 획득하세요.",
+  "advancement.mythicmetals.banglum_chunk.title": "폭발하는 바람에서 나온 덩어리",
+  "advancement.mythicmetals.bronze.description": "구리와 주석을 합친 클래식한 합금인 청동을 단조하세요.",
+  "advancement.mythicmetals.bronze.title": "청동 시대의 붕괴",
+  "advancement.mythicmetals.carmot.description": "심층암 층 상단 주변에서 카못을 찾으세요.",
+  "advancement.mythicmetals.carmot.title": "연금술의 기적",
+  "advancement.mythicmetals.carmot_staff.description": "안에 있는 블록에 따라 능력이 바뀌는 카못 지팡이를 만들기 위해 카못 블록을 사용하세요.",
+  "advancement.mythicmetals.carmot_staff.title": "카못 vs 세계",
+  "advancement.mythicmetals.celestium.description": "스타 플래티넘, 카이버, 언옵테이늄을 이용해 셀레스티움을 단조하세요.",
+  "advancement.mythicmetals.celestium.title": "멈출 수 없는 힘",
+  "advancement.mythicmetals.durasteel.description": "쿼드릴룸과 망간을 이용해 더 좋은 강철인 듀라스틸을 단조하세요.",
+  "advancement.mythicmetals.durasteel.title": "아주 아주 먼 은하계에서...",
+  "advancement.mythicmetals.forge.description": "여러 원석 광석을 녹여 더 많은 주괴를 얻을 수 있는 1티어 용광로 제어 장치를 제작하세요.",
+  "advancement.mythicmetals.forge.title": "영광스러운 합금 용광로",
+  "advancement.mythicmetals.forgery.description": "강력한 합금을 단조하는 데 사용할 수 있는 2티어 용광로 제어 장치를 제작하세요.",
+  "advancement.mythicmetals.forgery.title": "이게 용광로였다고 생각했어요",
+  "advancement.mythicmetals.gilded_midas_gold_sword.description": "더 강력한 황금의 힘을 지닌 검인 금으로 감싸 황금이 박힌 미다스 골드 검을 얻으세요.",
```

Comment:
I translated 'Gilded' to '황금이 박힌' because official minecraft's korean translation is '황금이 박힌'.
I'll change it to '더 강력한 황금의 힘을 지닌 도금된 미다스 골드 검을 단조하세요.'